test/extensions/filters/common/local_ratelimit/local_ratelimit_test.cc

```diff
@@ -516,10 +530,6 @@ TEST_F(LocalRateLimiterDescriptorImplTest, TokenBucketDifferentDescriptorStatus)
   // Limited by global token
```

Comment:
I think this comment is also wrong?

Reply:
You are right.

Actually, besides the comment, I think the [two validations right after it](https://github.com/envoyproxy/envoy/blob/fa34d0cb70af5ce38383b12f2026c389261b7001/test/extensions/filters/common/local_ratelimit/local_ratelimit_test.cc#L531) will not be necessary since the same checks are performed [a little further down](https://github.com/envoyproxy/envoy/blob/fa34d0cb70af5ce38383b12f2026c389261b7001/test/extensions/filters/common/local_ratelimit/local_ratelimit_test.cc#L539).

```c++
// Limited by global token
EXPECT_FALSE(rate_limiter_->requestAllowed(descriptor_));
EXPECT_EQ(rate_limiter_->maxTokens(descriptor_), 2);

// 0 -> 0 tokens for descriptor_ and descriptor2_
EXPECT_FALSE(rate_limiter_->requestAllowed(descriptor2_));
...
EXPECT_FALSE(rate_limiter_->requestAllowed(descriptor_));
EXPECT_EQ(rate_limiter_->maxTokens(descriptor_), 2);
```